### DISCUSSION TITLE
how to convert kml file to osm file using c# code

### DISCUSSION THREAD

**Discussion by VinayakPattar1995** [dotnet/csharplang#7437] [General]
Hi Everyone,
i want to convert the KML file to osm file by using c# code only (without using any third party tool) with all the information.

Please help me to convert.

**Comment by DavidArno** [comment]
Hi @VinayakPattar1995,

This discussion area is for discussing new language feature ideas, not for helping with writing C# code to solve specific problems.

You may be able to get help on this via https://stackoverflow.com/questions/ask. But be aware that you'll need to add more detail, show what you have done so far and where specifically you are stuck otherwise your question will likely just be closed on StackOverflow.

**Reply by VinayakPattar1995** [reply]
Thanks a lot dear....!
